x-pack/platform/plugins/private/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/indices/resolution_table_cell.tsx

```diff
@@ -67,12 +67,24 @@ const i18nTexts = {
   reindexText: i18n.translate('xpack.upgradeAssistant.esDeprecations.indices.reindexLabel', {
     defaultMessage: 'Reindex',
   }),
+  reindexFollowerIndexText: i18n.translate(
+    'xpack.upgradeAssistant.esDeprecations.indices.reindexFollowerIndexLabel',
+    {
+      defaultMessage: 'Unfollow leader index',
+    }
+  ),
   reindexTooltipLabel: i18n.translate(
     'xpack.upgradeAssistant.esDeprecations.indices.reindexTooltipLabel',
     {
       defaultMessage: 'Resolve this issue by reindexing into a new, compatible index.',
     }
   ),
+  reindexFollowerIndexTooltipLabel: i18n.translate(
+    'xpack.upgradeAssistant.esDeprecations.indices.reindexFollowerIndexTooltipLabel',
+    {
+      defaultMessage: 'Resolve this issue by unfollowing the leader index.',
```

Comment:
>       defaultMessage: 'Resolve this issue by unfollowing the leader index.',

Per my other comments, since it seems like there's multiple steps involved (i.e. "pause the replication, close the index, unfollow, and reopen the index") I'd recommend using the terminology "terminate replication" to align with (and link to) the docs.